implementations/rust/ockam/ockam_command/src/util/mod.rs

```diff
@@ -534,3 +539,24 @@ pub async fn query_pid(
     tx.send(Some(status.pid)).unwrap();
     Ok(())
 }
+
+#[test]
+fn test_extract_node_name() {
+    let test_cases = vec![
+        ("/node/test", "test", false),
+        ("test", "test", false),
+        ("/test", "/test", false),
+        ("/node/", "", false),
```

Comment:
I think this case should return an error instead of an empty string. Maybe you can add a "non_empty" check before returning the string.

A couple of test cases you could add:
- `"/node"`: I think this should fail when casting the `Node` value (at line 465)
- `""`: the other empty string case